### PR TITLE
JESI-2183

### PR DESCRIPTION
**Pull request by axrs**
[JESI-2183](https://jesims.atlassian.net/browse/JESI-2183)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -59,8 +74,9 @@ ENV LEIN_ROOT 1
 
 # Install clojure 1.9.0 so users don't have to download it every time
 RUN echo '(defproject dummy "" :dependencies [[org.clojure/clojure "1.9.0"]])' > project.clj \
-  && lein deps && rm project.clj
+	&& lein deps && rm project.clj
 
+#TODO: Kill it with fire
```
Comment (AndreTheHunter) [inline Dockerfile]: 🔥🔥🔥

File: Dockerfile [hunk 2]
```diff
@@ -35,62 +39,50 @@ RUN apk add --verbose --update --upgrade --no-cache \
...
-  && cd circleci-tools \
-  && git pull \
-  && chmod +x ./cancel-redundant-builds.sh
+	&& cd circleci-tools \
+	&& git pull \
+	&& chmod +x ./cancel-redundant-builds.sh
 ENV PATH=$PATH:/tmp/circleci-tools/
 RUN node -v > .node_version
+
+CMD ["bash"]
```
Comment (axrs) [inline Dockerfile]: Does this still work for circle?

Comment (AndreTheHunter) [reply Dockerfile]: CMD is overridable, (see https://github.com/nodejs/docker-node/blob/master/10/alpine/Dockerfile) and seems to work fine on CircleCI

Comment (axrs) [reply Dockerfile]: Cool.

**Pull request merged**